sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -156,8 +155,8 @@ export class AzureCliCredential implements TokenCredential {
         (err as Error).message || "Unknown error while trying to retrieve the access token"
       );
       span.setStatus({
```

Comment:
You could reduce the boilerplate here significantly if you used `withSpan`

Reply:
I’ll do the update, however:

I’m not sure how much boilerplate this reduces in some cases. Where the boilerplate reduced is little, wouldn’t the performance matter? since one needs to throw an extra time?

Reply:
Good point... I'm honestly not sure so I'll defer to you as package owner

Reply:
I’ll do the changes 🙂

Reply:
Decided to apply the changes to reduce noise in the future. Someone looking at the packages and asking why this code is one way or another might find the difference odd. We also don’t have sufficient data to say that withSpan is much less performant.